service-commands/src/setup.js

```diff
@@ -554,83 +588,101 @@ const identityServiceUp = async (options = { verbose: false }) => {
-    [Service.SOLANA_VALIDATOR, SetupCommand.UP]
+    [Service.SOLANA_VALIDATOR, SetupCommand.UP],
+    [Service.SOLANA_VALIDATOR, SetupCommand.HEALTH_CHECK]
   ]
 
-  const inParallel = [
+  const ipfsAndContractsCommands = [
     [Service.IPFS, SetupCommand.UP],
     [Service.IPFS_2, SetupCommand.UP],
     [Service.CONTRACTS, SetupCommand.UP],
     [Service.ETH_CONTRACTS, SetupCommand.UP],
-    [Service.SOLANA_PROGRAMS, SetupCommand.UP]
+    [Service.SOLANA_PROGRAMS, SetupCommand.UP],
   ]
 
-  let creatorNodeCommands = _.range(1, numCreatorNodes + 1).map(serviceNumber => {
-    return [
-      [
-        Service.CREATOR_NODE,
-        SetupCommand.UPDATE_DELEGATE_WALLET,
-        { serviceNumber, ...options }
-      ],
-      [
-        Service.CREATOR_NODE,
-        SetupCommand.UP,
-        { serviceNumber, ...options, waitSec: 10 }
-      ],
-      [
-        Service.CREATOR_NODE,
-        SetupCommand.HEALTH_CHECK,
-        { serviceNumber, ...options }
-      ],
-      [
-        Service.CREATOR_NODE,
-        SetupCommand.REGISTER,
-        { serviceNumber, ...options }
+  let creatorNodesCommands = _.range(1, numCreatorNodes + 1).map(
+    serviceNumber => {
+      return [
+        [
+          Service.CREATOR_NODE,
+          SetupCommand.UPDATE_DELEGATE_WALLET,
+          { serviceNumber, ...options }
+        ],
+        [
+          Service.CREATOR_NODE,
+          SetupCommand.UP,
+          { serviceNumber, ...options, waitSec: 10 }
+        ],
+        [
+          Service.CREATOR_NODE,
+          SetupCommand.HEALTH_CHECK,
+          { serviceNumber, ...options }
+        ],
+        [
+          Service.CREATOR_NODE,
+          SetupCommand.REGISTER,
+          { serviceNumber, ...options }
+        ]
       ]
-    ]
-  })
+    }
+  )
 
-  let discoveryNodesCommands = _.range(1, numDiscoveryNodes + 1).map(serviceNumber => {
-    return [
-      [
-        Service.DISCOVERY_PROVIDER,
-        SetupCommand.UP,
-        { serviceNumber, ...options }
-      ],
-      [
-        Service.DISCOVERY_PROVIDER,
-        SetupCommand.HEALTH_CHECK,
-        { serviceNumber, ...options }
-      ],
-      [
-        Service.DISCOVERY_PROVIDER,
-        SetupCommand.REGISTER,
-        { retries: 2, serviceNumber, ...options }
+  let discoveryNodesCommands = _.range(1, numDiscoveryNodes + 1).map(
+    serviceNumber => {
+      return [
+        [
+          Service.DISCOVERY_PROVIDER,
+          SetupCommand.UP,
+          { serviceNumber, ...options }
+        ],
+        [
+          Service.DISCOVERY_PROVIDER,
+          SetupCommand.HEALTH_CHECK,
+          { serviceNumber, ...options }
+        ],
+        [
+          Service.DISCOVERY_PROVIDER,
+          SetupCommand.REGISTER,
+          { retries: 2, serviceNumber, ...options }
+        ]
       ]
-    ]
-  })
+    }
+  )
 
   const sequential1 = [
     [Service.INIT_CONTRACTS_INFO, SetupCommand.UP],
+    [Service.USER_REPLICA_SET_MANAGER, SetupCommand.UP],
```

Comment:
sequential1 here is just a cluster of random weird stuff. i think fine for now, but i'd like to rethink where all of this misc. local setup follow on things should live